lib/ex_geo/store.ex

```diff
@@ -19,14 +19,19 @@ defmodule ExGeo.Store do
 
   def init(_) do
     send(self(), :download)
-    {:ok, []}
+    {:ok, :ets.new(:exgeo_store, [:named_table, :set, :protected])}
```

Comment:
oh this is a perfect case for https://hexdocs.pm/elixir/GenServer.html#c:handle_continue/2 I think

Reply:
Yeah but this way we can reuse the update func